core/message-tags-3.3.md

```diff
@@ -135,9 +135,14 @@ Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client
 ## Security considerations
 
 Client-only tags should be treated as untrusted data. They can contain any value
-and are not validated by servers in any way.
+and are not validated by servers in any way. The server MAY normalise values by applying the unescape
+and escape process.
```

Comment:
Hmm, I'm a little confused by how/when the server would apply the escape/unescape process itself. It seems like allowing this would just let servers randomly break incoming tagged messages by incorrectly applying escaping where it shouldn't exist?

Is this intended so that servers can essentially fix cases where clients have accidentally left a tag value as unescaped where they meant to escape it? If so then this seems like a fragile fix that'll have to depend on heuristically working out when to apply it.